the-collab-lab/tcl-70-smart-shopping-list
# Issue #13 | Sort list by purchase urgency

## Description

- Sorts items within a selected list by how soon the item should be purchased
- Adds text to the UI to indicate how soon the item should be puchased

## Related Issue

closes #13 

## Acceptance criteria


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: Enhancement    |


## Updates

### Before

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/ef10ffc4-5c00-4ec2-8357-3a2a0ae67c41)

### After

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/a418d6d0-c451-4fc3-abb9-81adf2e494a2)

## Testing Steps / QA Criteria

1. From your terminal, pull down this branch with `git pull origin ag-km-sort-list` and check that branch out with `git checkout ag-km-sort-list`
2. Sign into the app and select a list
3. Click the link at the bottom of the page to navigate to the Manage List page
4. Add two or more new items of different purchase urgencies
5. Navigate back to the List page to view how the new items are sorted. 
6. Items that should be bought "soon" should be at the top of the list, followed by  item that should be bought "kind of soon", then those that are labeled "not soon". All items that are labeled "inactive" should be at the bottom of the list.
